.storybook/preview.js

```diff
@@ -11,6 +10,9 @@ export const parameters = {
       color: /(background|color)$/i,
       date: /Date$/,
     },
+    viewport: {
+      defaultViewport: 'Mobile'
+    }
```

Comment:
nice!

Reply:
Actually this isn't working and i just forgot to delete it.